src/main/java/songbox/house/service/BandcampService.java

```diff
@@ -0,0 +1,11 @@
+package songbox.house.service;
+
+import songbox.house.domain.dto.response.SearchAndDownloadResponseDto;
+//import songbox.house.util.Measurable;
+
+import java.util.Set;
+
+public interface BandcampService {
+//    @Measurable
```

Comment:
Looks disabled?

Reply:
Yes, unfortunately, for now, this annotation doesn't work with interface methods. It works only with `public` methods in the `implementation`